modules/fetchEventData.js

```diff
@@ -0,0 +1,108 @@
+	
+	
+
+    function syncEventData() {
+      	let isTimeStamp = kony.store.getItem("isTimeStampUpdated");
+      	if(isTimeStamp === null || isTimeStamp === undefined)
+      		kony.store.setItem("clientLastUpdatedTime","2019-08-04T15:27:26Z");
+        fetchObjectData(eventConstants.OBJECT_SERVICE_NAME, eventConstants.DATA_SYNC_OBJECT, {}, dataSyncFetchSuccess, dataSyncFetchFailure);
+    }
+ 
+    function dataSyncFetchSuccess(successResponse) {
+        let records = (successResponse.hasOwnProperty("records")) ? successResponse.records : null;
+        if (records !== null) {
```

Comment:
Best practice is to keep constant in left in compare operators.
```if (null !=== records) {```

This is applicable for whole